tests/test_pyclipper.py

```diff
@@ -246,8 +246,8 @@ def test_execute2(self):
 
     def test_clear(self):
         self.pc.Clear()
-        solution = self.pc.Execute(*self.default_args)
-        self.assertEqual(len(solution), 0)
+        with self.assertRaises(pyclipper.ClipperException):
+            solution = self.pc.Execute(*self.default_args)
```

Comment:
Also the return value is unused, so you could just call Execute without saving solution, but it’s ok.